src/match.ts

```diff
@@ -36,16 +36,16 @@ export class Matcher {
    *
    * @param network - An address or subnet
    *
-   * @returns A boolean
+   * @returns A Network or null
```

Comment:
It's a bit odd that the method named `has` returns a non-boolean value. Makes it less intuitive as generally a method with that name would be expected to return a boolean.

Could it be a new method instead? For example, `getNetwork`.

Reply:
Yeah agreed. Have updated this.